test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1149,20 +1189,29 @@ void UcxConnection::established(ucs_status_t status)
     invoke_callback(_establish_cb, status);
 }
 
-bool UcxConnection::send_common(const void *buffer, size_t length, ucp_tag_t tag,
+bool UcxConnection::send_common(const void *buffer, ucp_mem_h memh,
+                                size_t length, ucp_tag_t tag,
                                 UcxCallback* callback)
 {
+    ucp_request_param_t params = {};
```

Comment:
no need to initialize

Reply:
Can it pass coverity w/o the initialization?

Reply:
there is other function where it's not initialized